spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/MetricsProperties.java

```diff
@@ -342,4 +352,31 @@ public AutoTimeProperties getAutotime() {
 
 	}
 
+	public static class System {
+
+		private final DiskSpace diskspace = new DiskSpace();
+
+		public DiskSpace getDiskspace() {
+			return this.diskspace;
+		}
+
+		public static class DiskSpace {
```

Comment:
I still think it'd be good to rename this to `Diskspace`. That'd be consistent with what we've done elsewhere. See `org.springframework.boot.autoconfigure.web.WebProperties.Resources.Cache.Cachecontrol`, for example.